src/SprykerEco/Zed/Unzer/Business/Import/Filter/UnzerPaymentMethodImportFilterInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Import\Filter;
+
+use ArrayObject;
+
+interface UnzerPaymentMethodImportFilterInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\PaymentMethodTransfer|\ArrayObject $paymentMethodTransfers
+     * @param \Generated\Shared\Transfer\PaymentMethodTransfer|\ArrayObject $storedPaymentMethodTransfers
+     *
+     * @return \ArrayObject
```

Comment:
```suggestion
     * @param \ArrayObject<\Generated\Shared\Transfer\PaymentMethodTransfer> $paymentMethodTransfers
     * @param \ArrayObject<\Generated\Shared\Transfer\PaymentMethodTransfer> $storedPaymentMethodTransfers
     *
     * @return \ArrayObject<\Generated\Shared\Transfer\PaymentMethodTransfer>
```

Fix this in a `UnzerPaymentMethodImportImportFilter` class as well, please

Reply:
Fixed